packages/CollabCodeEdit-Client.package/CCEClient.class/instance/sendMessage..st

```diff
@@ -0,0 +1,4 @@
+sending
+sendMessage: aCCEMessage
+	Transcript show: aCCEMessage serialize.
```

Comment:
Debug code should not be pulled onto develop

Reply:
If i remember correctly, we already got `Transcript show:` in the develop, but your absolutely right.